sklearn/preprocessing/tests/test_label.py

```diff
@@ -35,16 +36,26 @@ def toarray(a):
 
 
 def test_label_binarizer():
-    lb = LabelBinarizer()
-
     # one-class case defaults to negative label
```

Comment:
Is this comment appropriate? I'd rather delete it as the test seems self explanatory...